File: REVINPUT.php
<?php
fscanf(STDIN,"%d",$n);
$l=trim(fgets(STDIN));
$p=explode(" ",$l);
$i=count($p)+1;
do
{
$k=strrev($p[$i]);
for($x=0;$x<$n;$x++)
echo $k;
$i--;
}while($i!=0);
?>

File: REVINPUT.py
import sys
t,f,s=0,0,[]
for _ in sys.stdin:
 if f==0:t=int(_);f=1;continue
 s.append(_.split())
for j in s[::-1]:
 for i in j[::-1]:print(i[::-1]+' ')*t,
